js/lib/components/feedback/selection.tsx

```diff
@@ -0,0 +1,98 @@
+import { VirtualElement } from "@popperjs/core";
+import React, { useCallback, useEffect, useState } from "react";
+import Highlighter from "web-highlighter";
+
+import { HIGHLIGHT_STORAGE_KEY } from "../../entryPoints/feedback";
+import FeedbackModal from "./modal";
+import SelectionTooltip from "./tooltip";
+
+type SelectionRendererProps = { highlighter: Highlighter; stored: any[] | undefined };
+let SelectionRenderer: React.FC<SelectionRendererProps> = ({ highlighter, stored }) => {
```

Comment:
Is there a reason that you show the feedback `textarea` in a modal instead of a tooltip? Seems like you could reuse the component rather than needing a distinct visual element.

Reply:
Yeah I experimented with this but found the textarea resize interaction *very* weird (see video), although resizing isn't a requirement. It would also require some custom click location logic a la the method described in [this comment](https://github.com/willcrichton/mdbook-quiz/pull/5#discussion_r936227715), since we'd have to check if a click or selection is within or outside the tooltip to determine whether or not to close it (as opposed to react-modal which has built-in callbacks for actions that close the modal). Sorry if this isn't making a whole lot of sense - I can try to clear it up tomorrow.

https://user-images.githubusercontent.com/40175891/182528899-49048abb-be9b-4b3d-9270-84d5b038f157.mp4